docs/lib/datastore/fragments/flutter/relational/save-snippet.md

```diff
@@ -1,3 +1,16 @@
 ```dart
-TBD
+Post post = Post(
+    title: "My Post with comments", rating: 10, status: PostStatus.ACTIVE);
+Comment comment = Comment(
+    post: post, // Directly pass in the post instance
+    content: "Loving Amplify DataStore!");
+
+try {
+  await Amplify.DataStore.save(post);
+  print("Post saved");
+  await Amplify.DataStore.save(comment);
+  print("Comment saved");
+} catch (e) {
```

Comment:
Is there a way to remove these `catch` statements from the examples, for brevity? Maybe you can add an `.onError` hook to the end of a functional chain?

Reply:
Removed catch statements on the relational data access snippets